### PR TITLE
Corrected code styling and added missing Write-Host mocks

### PR DESCRIPTION
**Pull request by ykuijs**
#### Pull Request (PR) description
This PR corrects code styling issues in all DSC resources and adds missing mocks for Write-Host in several unit tests, so the test output will be clean.

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

File: Modules/Microsoft365DSC/DSCResources/MSFT_AADGroup/MSFT_AADGroup.psm1 [hunk 1]
```diff
@@ -1016,8 +1017,8 @@ function Export-TargetResource
     try
     {
         [array] $groups = Get-MgGroup -Filter $Filter -All:$true -ErrorAction Stop
-        $groups = $groups | Where-Object -FilterScript {-not ($_.MailEnabled -and ($null -eq $_.GroupTypes -or $_.GroupTypes.Length -eq 0)) -and
-                                                        -not ($_.MailEnabled -and $_.SecurityEnabled)}
+        $groups = $groups | Where-Object -FilterScript { -not ($_.MailEnabled -and ($null -eq $_.GroupTypes -or $_.GroupTypes.Length -eq 0)) -and
```
Comment (andikrueger) [inline Modules/Microsoft365DSC/DSCResources/MSFT_AADGroup/MSFT_AADGroup.psm1:1020]: Wouldn't it be better to add a proper line break here: `

**Review by andikrueger — COMMENTED** [review]
Just some minor questions.

File: Modules/Microsoft365DSC/DSCResources/MSFT_PPTenantIsolationSettings/MSFT_PPTenantIsolationSettings.psm1 [hunk 2]
```diff
@@ -718,7 +718,7 @@ function Get-M365TenantId
         $TenantName
     )
 
-    if ($TenantName -notmatch ".onmicrosoft.com$")
+    if ($TenantName -notmatch '.onmicrosoft.com$')
```
[the remark this answers is not in the document]
Comment (ykuijs) [reply Modules/Microsoft365DSC/DSCResources/MSFT_PPTenantIsolationSettings/MSFT_PPTenantIsolationSettings.psm1:721]: Yes, with regular expressions this mean "at the end of the line"

File: Modules/Microsoft365DSC/DSCResources/MSFT_AADGroup/MSFT_AADGroup.psm1 (hunk 1 above)
Comment (ykuijs) [reply Modules/Microsoft365DSC/DSCResources/MSFT_AADGroup/MSFT_AADGroup.psm1:1020]: Done!

**Pull request merged**